src/main/java/command/ListCommand.java

```diff
@@ -0,0 +1,38 @@
+package command;
+
+import duke.DukeException;
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import task.Task;
+
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+import static java.lang.String.format;
+
+/**
+ * Command to List the existing tasks.
+ */
+public class ListCommand extends Command {
+
+    /**
+     * Command to List the existing tasks.
+     */
+    public ListCommand() {
+        this.setExit(false);
```

Comment:
Instead if set `isExit` to be `false` in every `Command` subclass (except for `ByeCommand`), an alternative implementation is to  default `isExit` to be false in the parent `Command` class and then you only need to set it to be true in `ExitCommand` class.